packages/typespec-test/test/eventgrid_modular/generated/typespec-ts/review/eventgrid.api.md

```diff
@@ -42,12 +47,12 @@ export interface CloudEvent {
 // @public (undocumented)
 export class EventGridClient {
     constructor(endpoint: string, credential: KeyCredential, options?: EventGridClientOptions);
-    acknowledgeCloudEvents(lockTokens: string[], topicName: string, eventSubscriptionName: string, options?: AcknowledgeCloudEventsOptions): Promise<AcknowledgeResult>;
-    publishCloudEvent(event: CloudEvent, topicName: string, options?: PublishCloudEventOptions): Promise<Record<string, any>>;
-    publishCloudEvents(events: CloudEvent[], topicName: string, options?: PublishCloudEventsOptions): Promise<Record<string, any>>;
+    acknowledgeCloudEvents(topicName: string, eventSubscriptionName: string, lockTokens: AcknowledgeOptions, options?: AcknowledgeCloudEventsOptions): Promise<AcknowledgeResult>;
```

Comment:
What are the root causes of those parameter order change?

Reply:
This is expected, because previously it will put the flattened parameters ahead of other required parameters.